src/Task/PhpMnd.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace GrumPHP\Task;
+
+use GrumPHP\Runner\TaskResult;
+use GrumPHP\Task\Context\ContextInterface;
+use GrumPHP\Task\Context\GitPreCommitContext;
+use GrumPHP\Task\Context\RunContext;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * PhpMnd task
+ */
+class PhpMnd extends AbstractExternalTask
+{
+    /**
+     * @return string
+     */
+    public function getName()
+    {
+        return 'phpmnd';
+    }
+
+    /**
+     * @return OptionsResolver
+     */
+    public function getConfigurableOptions()
+    {
+        $resolver = new OptionsResolver();
+        $resolver->setDefaults([
+            'directory' => '.',
```

Comment:
Nice catch! I looked into it and it turns out that PHPMND automatically adds `vendor` to the exclude list ([as seen here](https://github.com/povils/phpmnd/blob/a85c7c746c52f15b8326596523889b16b39a7665/src/PHPFinder.php#L32)), so it should be fine as-is.